SConstruct

```diff
@@ -248,6 +248,8 @@ condition_map = {
     'POSIX_HOST' : env['PLATFORM'] == 'posix',
     'POWERPC_HOST' : False,
     'WINDOWS_HOST' : env['PLATFORM'] == 'win32',
+    'X86_HOST' : True,
+    'ZSERIES_HOST' : False,
```

Comment:
FTR, we'll need to make the corresponding changes to MongoDB's `SConscript` file for WiredTiger before this change can be merged.